### ISSUE TITLE
Piles don't inherit movable

### ISSUE THREAD

**Issue by ArnoldSmith86** [ArnoldSmith86/virtualtabletop#105]
If cards are not movable, piles still are.

**Comment by robartsd** [comment]
I think the solution for cards that are not movable would be to disallow them from becoming part of a pile when another card is dropped on them. I've also noticed that piles do not inherit the grid of their children.

**Comment by 96LawDawg** [comment]
I think this fits here.  PR#87 broke something else related to piles.  I had to recode my move view/eye shift because it relied on the old piles concept.  I have figured it out, BUT, the pile handles do not move with the pile.  See the image below.  The 0-2 domino was part of a 2-card pile that was shifted down.  The pile handle (with the #2) remained behind.  Sample room at https://virtualtabletop.io/zopo

![Pile Handles](https://user-images.githubusercontent.com/76912527/104776762-3265fc00-5740-11eb-9714-a78febf48648.jpg)
